stable/msoms/README.md

```diff
@@ -0,0 +1,59 @@
+
+## Introduction
+
+This chart deploys an OMS daemonset on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager. The OMS agent enables rich and real-time analytics for Docker containers. With this solution, you can see which containers are running on your container hosts and what images are running in the containers. You can view detailed audit information showing commands used with containers. And, you can troubleshoot containers by viewing and searching centralized logs without having to remotely view Docker or hosts. You can find containers that may be noisy and consuming excess resources on a host. And, you can view centralized CPU, memory, storage, and network usage and performance information for containers. For more information refer to the [documentation](https://docs.microsoft.com/en-us/azure/log-analytics/log-analytics-containers).
+
+## Prerequisites
+
+- Kubernetes 1.6+
+- Add the Container Monitoring solution to the [Azure Log Analytics workspace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/microsoft.containersoms?tab=Overview).
+
+## Installing the Chart
+
+```bash
+$ helm install --name omsagent stable/msoms
+```
+
+## Uninstalling the Chart
+
+To uninstall/delete the `omsagent` deployment:
+
+```bash
+$ helm delete --purge omsagent
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following tables lists the configurable parameters of the MSOMS chart and their default values.
+
+| Parameter                  | Description                        | Default                                                    |
+| -----------------------    | ---------------------------------- | ---------------------------------------------------------- |
+| `omsagent.image.tag`       | `msoms` image tag.                 | Most recent release aka latest                             |
+| `omsagent.secret.wsid`     | OMS workspace id                   | Does not have a default value, needs to be provided        |
+| `omsagent.secret.key`      | OMS workspace key                  | Does not have a default value, needs to be provided        |
+| `omsagent.domain`          | OMS cloud domain (public / govt)   | opinsights.azure.com (Public cloud as default), opinsights.azure.us (Govt Cloud) |
+
+To get your workspace id and key do the following
+- In the Azure Log Analytics portal, on the Overview page, click the Settings tile. Click the Connected Sources tab at the left and select "Linux Server" on the right hand side.
```

Comment:
Would it be worth adding these as coments in values.yaml that way you can see them when running helm inspect?

Reply:
Specifically the key and the wsid

Reply:
Done